tests/sentry/web/frontend/test_create_organization_member.py

```diff
@@ -60,7 +60,7 @@ def test_valid_for_invites(self):
         with self.settings(SENTRY_ENABLE_INVITES=True), self.tasks():
             resp = self.client.post(path, {
                 'email': 'foo@example.com',
-                'role': 'member',
+                'role': 'admin',
```

Comment:
the only reason i changed the tests was because the default is member, so even though it was broken, testing for member didn't actually catch the bug. i wanted to change it to something that wasn't the default, so if we regress on this, the tests will actually catch it.